statsmodels/tsa/statespace/mlemodel.py

```diff
@@ -159,6 +164,11 @@ def fit(self, start_params=None, transformed=True,
         return_params : boolean, optional
             Whether or not to return only the array of maximizing parameters.
             Default is False.
+        hessian_method : {'opg','oim','cs'}, optional
+            The method by which the Hessian is numerically approximated. 'opg'
+            uses outer product of gradients, 'oim' uses the information
+            matrix formula from Harvey (1989), and 'cs' uses second-order
```

Comment:
Users can set the hessian method because there are three approximations to the Hessian available, and from BHHH (e.g. p655) we need to use a positive definite matrix that approximates the Hessian. In some cases, I've found the `cs` Hessian approximation may not be positive definite.

So to my understanding, there are two uses for the Hessian here:

1. In optimization (i.e. choosing the direction and stepsize in the optimizer), and that can be affected via `hessian_method`.
2. In inference, and that can be affected via `cov_type`.

We don't have to have the first one, and I can just eliminate `hessian_method`.